lib/alephant/broker/load_strategy/http.rb

```diff
@@ -22,7 +22,10 @@ def initialize(url_generator)
         def load(component_meta)
           cache_object(component_meta)
         rescue
-          logger.metric(:name => "BrokerLoadStrategyHTTPCacheMiss", :unit => "Count", :value => 1)
+          logger.metric(
+            "BrokerLoadStrategyHTTPCacheMiss",
```

Comment:
Are we not removing the additional meta we have encoded in the metric name `BrokerLoadStrategyHTTP` as this is captured in the dimension itself.